src/main.rs

```diff
@@ -107,7 +107,7 @@ impl Future for MainLoopState {
                                          audio_filter,
                                          move || (backend)(audio_device));
 
-                let (spirc, spirc_task) = Spirc::new(session, player, (self.mixer)());
+                let (spirc, spirc_task) = Spirc::new("Spotifyd".to_string(), session, player, (self.mixer)());
                 self.spirc_task = Some(spirc_task);
```

Comment:
Perhaps we can add a `device_name: String` to `MainLoopState`, and then use it here instead of hard coding?